src/BotanDB.php

```diff
@@ -32,8 +32,8 @@ public static function initializeBotanDb()
     /**
```

Comment:
I think it would make sense to use the same parameter order like for `shortenUrl($url, $user_id)`:
so `selectShortUrl($url, $user_id)`